desc/objectives/linear_objectives.py

```diff
@@ -1689,6 +1819,173 @@ def compute(self, Z_lmn, **kwargs):
         return f
 
 
+class FixSumModesLambda(_FixedObjective):
```

Comment:
I think this shouldn't inherit from `_FixedObjective` because it doesn't give a diagonal constraint matrix

Reply:
hm, so then should the FixSumModesR and FixSumModesZ also not be inheriting?

Reply:
wait nvm this is fine, the `_FixedObjective` base class really just defines the `update_target` method